pkg/migrations/op_add_column_test.go

```diff
@@ -150,7 +150,25 @@ func TestAddColumn(t *testing.T) {
 					},
 				},
 			},
-			wantStartErr: backfill.NotPossibleError{Table: "users"},
+			afterRollback: func(t *testing.T, db *sql.DB, schema string) {
+				// The new column has been dropped from the underlying table
+				columnName := migrations.TemporaryName("age")
+				ColumnMustNotExist(t, db, schema, "users", columnName)
+
+				// The table's column count reflects the drop of the new column
+				TableMustHaveColumnCount(t, db, schema, "users", 1)
+
+				// Insert a row to make sure we get a not possible error because there's no identity column
```

Comment:
```suggestion
				// Insert a row to make sure we get a `backfill.NotPossible` error when the migration is restarted, because there's no identity column
```